doc/manual/hacking.xml

```diff
@@ -4,18 +4,31 @@
 
 <title>Hacking</title>
 
-<para>This section provides some notes on how to hack on Nix.  To get
+<para>This section provides some notes on how to hack on Nix. To get
 the latest version of Nix from GitHub:
 <screen>
-$ git clone git://github.com/NixOS/nix.git
+$ git clone https://github.com/NixOS/nix.git
 $ cd nix
 </screen>
 </para>
 
-<para>To build it and its dependencies:
+<para>To build it Nix using one of the targets provided by <literal>default.nix</literal>:
 <screen>
-$ nix-build release.nix -A build.x86_64-linux
+$ nix-build -A outputs.hydraJobs.build.x86_64-linux
+$ nix-build -A outputs.hydraJobs.build.aarch64-linux
+$ nix-build -A outputs.hydraJobs.build.x86_64-darwin
+$ nix-build -A outputs.hydraJobs.build.i686-linux
 </screen>
+
+or if you have a flakes-enabled nix:
+
+<screen>
+$ nix build ".#hydraJobs.build.x86_64-linux"
```

Comment:
Worth mentioning that you can do `nix build` without any arguments to get a build for the current platform.

Reply:
Simplified.